pypeit/tests/test_runpypeit.py

```diff
@@ -128,10 +128,10 @@ def test_run_pypeit():
     # #########################################################
     # Test!!
     # Files exist
-    assert os.path.isfile(os.path.join(configdir, 'Science', 'spec2d_b27-J1217p3905_KASTb_2015May20T045733.560.fits'))
```

Comment:
I am sure that are many more file names in the tests that need to be changed.

Reply:
I think they will all be in the Dev suite from here (all unit tests are passing)

Reply:
@profxj what I meant here is that there are other unit tests that read in cooked spec1d or spec2d files that in the unit test have still the old name format.

For example in [test_slitmask.py](https://github.com/pypeit/PypeIt/blob/0812fb4fd3d2a00c0c679d90ef2c4d53b973aa98/pypeit/tests/test_slitmask.py) the method `test_assign_maskinfo` calls a cooked spec1d: 
```
 specobjs_file = os.path.join(os.getenv('PYPEIT_DEV'), 'Cooked', 'Science',
                            'spec1d_DE.20100913.22358-CFHQS1_DEIMOS_2010Sep13T061231.334.fits')
```
And I am guessing that after running the Dev Suite and created new cooked, this test will fail.